public/js/main.js

```diff
@@ -1,222 +1,193 @@
+            labels: ['White', 'Black', 'Asian', 'Multiple Races', 'Other', 'American Indian', 'Pacific Islander'],
+            datasets: [
+                {
+                    backgroundColor: '#316BBF',
+                    data: [982511, 532025, 57948, 57205, 27772, 6790, 2059]
+                }
+            ]
+        };
+
+        new Chart($('#populationByRace'), {
+            type: 'horizontalBar',
+            data: populationByRace
+        })
     });
 
-    var polarData = [
-        {
-            value: 300,
-            color: '#F7464A',
-            highlight: '#FF5A5E',
-            label: 'Red'
-        },
-        {
-            value: 50,
-            color: '#46BFBD',
-            highlight: '#5AD3D1',
-            label: 'Green'
-        },
-        {
-            value: 100,
-            color: '#FDB45C',
-            highlight: '#FFC870',
-            label: 'Yellow'
-        },
-        {
-            value: 40,
-            color: '#949FB1',
-            highlight: '#A8B3C5',
-            label: 'Grey'
-        },
-        {
-            value: 120,
-            color: '#4D5360',
-            highlight: '#616774',
-            label: 'Dark Grey'
-        }
-
-    ];
-
+    // School rating by enrollment
     $(document).ready(function () {
-        window.myPolarArea = new Chart(document.getElementById('canvas_area').getContext('2d')).PolarArea(sharePiePolorDoughnutData, {
-            responsive: true,
-            tooltipFillColor: 'rgba(51, 51, 51, 0.55)'
+        var ratingByEnrollment;
+
+        // Gather aggregates from Hercules
+        $.ajax({
+            method: 'GET',
+            url: 'http://hercules.code4hr.org/schools?groupBy=enrollment&averageField=gsRating',
+            data: ratingByEnrollment,
+            crossDomain: true,
+            dataType: 'jsonp',
+            success: function (ratingByEnrollment) {
+                var dataForChart = [];
+                ratingByEnrollment = ratingByEnrollment.data;
+                // Create data array for chart
+                for(var i = 0; i <= ratingByEnrollment.length; i++) {
+                    for(key in ratingByEnrollment[i]) {
+                        dataForChart.push({x: Number(ratingByEnrollment[i][key]), y: Number(key), r: 3});
+                    }
+                }
+
+                var data = {
+                    datasets: [
+                        {
+                            label: 'School Rating By Enrollment',
+                            data: dataForChart,
+                            backgroundColor: '#7715F7',
+                            hoverBackgroundColor: '#7715F7',
+                        }
+                    ]
+                };
+
+                var options = {
+                    scales: {
+                        yAxes: [{
+                            scaleLabel: {
+                                display: true,
+                                labelString: 'Number of Enrollment'
```

Comment:
Number of Enrolled Students doesn't fit on the axis.. How about Students Enrolled?